algoliasearch/src/offline/java/com/algolia/search/saas/OfflineClient.java

```diff
@@ -120,6 +124,32 @@ public MirroredIndex initIndex(@NonNull String indexName)
     }
 
     /**
+     * Obtain a mirrored index. Although this will always be an instance of {@link MirroredIndex}, mirroring is
+     * deactivated by default.
+     *
+     * @param indexName The name of the index.
+     * @return A proxy to the specified index.
+     */
+    @Override
+    public @NonNull MirroredIndex getIndex(@NonNull String indexName) {
+        MirroredIndex index = null;
+        WeakReference<Index> existingIndex = indices.get(indexName);
+        if (existingIndex != null) {
+            Index anIndex = existingIndex.get();
+            if (anIndex != null && !(anIndex instanceof MirroredIndex)) {
```

Comment:
Mmh... Currently, none. :D But in the future, we might have purely offline indices. I think I have anticipated too much. ;)